cinder/volume/drivers/lightos.py

```diff
@@ -27,6 +27,10 @@
 from oslo_utils import units
 import requests
 import urllib3
+try:
```

Comment:
why this is needed? python2 and python3?
in upstream they only use python3 -> can we remove the python2 option?

Reply:
ok